stb_vorbis.c

```diff
@@ -1469,7 +1511,7 @@ static int start_page_no_capturepattern(vorb *f)
       return error(f, VORBIS_unexpected_eof);
    // assume we _don't_ know any the sample position of any segments
    f->end_seg_with_known_loc = -2;
-   if (loc0 != ~0U || loc1 != ~0U) {
+   if (loc != ~(uint64)0U) {
```

Comment:
The `U` suffix is funny here. The constant can simply be `~((uint64)0)`

Reply:
Fixed.